Orso.Arpa.Application/AppointmentApplication/Interfaces/IAppointmentService.cs

```diff
@@ -5,38 +5,47 @@
 using Orso.Arpa.Application.AppointmentParticipationApplication.Model;
 using Orso.Arpa.Domain.AppointmentDomain.Enums;
 
-namespace Orso.Arpa.Application.AppointmentApplication.Interfaces
+namespace Orso.Arpa.Application.AppointmentApplication.Interfaces;
```

Comment:
Die ganzen Umformatierungen machen das Code Review extrem schwierig, weil man die echten Änderungen kaum erkennt. Warum ist das alles umformatiert? Wir haben eigentlich eine editorconfig, die die Formatierungen vorgibt. Bitte prüfe doch nochmal die Formatierungs-Einstellungen in deiner IDE

Reply:
ja, ist mir auch schon aufgefallen, als ich mir hier die files angeschaut habe, ich suche immer noch die Ursache dafür 🙈